flowcraft/templates/downsample_fastq.py

```diff
@@ -156,8 +157,8 @@ def main(sample_id, fastq_pair, genome_size, depth, clear, seed):
                     os.remove(rp)
 
     else:
-        os.symlink(p1, "{}._ss.fq.gz".format(bn1))
-        os.symlink(p2, "{}._ss.fq.gz".format(bn2))
+        shutil.copy(p1, "{}._ss.fq.gz".format(bn1))
+        shutil.copy(p2, "{}._ss.fq.gz".format(bn2))
```

Comment:
and here? I would recommend to do this in a different PR. Also in changelog this isn't reflected.

Reply:
added to changelog